### PR TITLE
Diagnostics: strategies for testing and practical considerations(e.g., reagents, PPE)

### PR DESCRIPTION
**Pull request by rando2**
Here is a first pass at adding these concerns from @cgreene to the outline. I'm planning to open a linked issue when we know what experts we want to try to bring in.

### CODE REVIEW

**Comment by rando2** [comment]
@juliettemarie0405 , is there a public health element here? Are there other areas of expertise you would recommend we bring in to address this question (I assume epidemiology?)

**Comment by cgreene** [comment]
Going to do the merge first.

**Comment by cgreene** [comment]
This was a pretty complex merge and I ended up hybridizing the sections more than I anticipated. Can you take a look at https://github.com/greenelab/covid19-review/pull/20/commits/0cac19baa588315917ee940fdadab50550df4cb6 and make sure it is aligned with what you intended?

File: content/10.diagnostics.md [hunk 1]
```diff
@@ -18,3 +18,13 @@ However, many countries have struggled to acquire the tests required to keep pac
...
 [What is sustained immunity and what are the indicators?]
+
+### Limitations to Implementation of Large-Scale Testing
+
+[Right now, reagent supply is an issue. Are there others concerns that are likely to emerge?]
+
+### Stratgies and Considerations for Determining Whom to Test 
+
+[If it's not possible to test everyone, what strategies exist for selecting who to test?]
+[Are these strategies likely to change over time? Preumably there are different stages of managing spread vs mitigating severity once it's already at high prevalence?]
```
Comment (cgreene) [inline content/10.diagnostics.md]:
```suggestion
[Are these strategies likely to change over time? Presumably there are different stages of managing spread vs mitigating severity once it's already at high prevalence?]
```

**Comment by cgreene** [comment]
Ok - you may want to just start back on your master and start a new branch. If your own master got some undesired things merged into it (that aren't in upstream), you may want to force reset it to match upstream before starting the next PR off of it. I'm guessing something like this happened.

**Comment by rando2** [comment]
@cgreene I think it's all fixed now! I got a little overwhelmed on Monday (when this commit was made) and I think my local copy got behind the upstream in ways that couldn't be easily overcome, but now it should all be good.

**Pull request merged**